include/basefunctions.hxx

```diff
@@ -376,7 +376,9 @@ inline auto FilterID(const int &index) {
 /// \returns a lambda function to be used in RDF Define
 inline auto FilterJetID(const int &index) {
     return [index](const ROOT::RVec<Int_t> &IDs) {
-        ROOT::RVec<int> mask = IDs == index;
+        ROOT::RVec<int> mask = IDs >= index;
```

Comment:
I think the reason why I did this is that the [recommendation](https://twiki.cern.ch/twiki/bin/view/CMS/JetID13TeVUL#Preliminary_Recommendations_for) is to use the lepVeto and then you have only one option, so there shouldn't be any difference. But it might generally be better to do it like this.

Reply:
I see, I thought it was set to `6`. Then I mixed something up I guess and this should be fixed. I think to use `>=2` should also be alright for us since we clean the jets from the leptons ourselves.